astropy/coordinates/distances.py

```diff
@@ -63,34 +62,40 @@ class Distance(u.SpecificTypeQuantity):
     ndmin : int, optional
         See `~astropy.units.Quantity`.
     allow_negative : bool, optional
-        Whether to allow negative distances (which are possible is some
-        cosmologies).  Default: ``False``.
+        Whether to allow negative distances (which are possible in some
+        cosmologies).  Default: `False`.
 
     Raises
     ------
     `~astropy.units.UnitsError`
```

Comment:
The `Distance` constructor no longer raises a `UnitsError` itself, but trying to use an invalid unit does lead to a `UnitsError` being raised elsewhere.

Reply:
I think it is OK to leave it here, though.